clang/test/AST/Interp/records.cpp

```diff
@@ -335,7 +335,7 @@ namespace InitializerTemporaries {
 
   constexpr int f() {
     S{}; // ref-note {{in call to 'S{}.~S()'}} \
-         // expected-note {{in call to '&S{}->~S()'}}
+         // expected-note {{in call to '~S()'}}
```

Comment:
This looks wrong in a different way now. Why is the instance missing?

Reply:
Good catch! It's 100% incorrect.